src/entry.h

```diff
@@ -109,6 +109,9 @@ void CacheEntryUnref(CacheEntry *self);
 CacheEntry *CacheEntryUnserialize(const gchar *value, GError **error);
 gchar *CacheEntrySerialize(CacheEntry *self);
 
+void CacheEntryHashSet(CacheEntry *self, CacheEntryAlgorithm alg,
+                  GBytes *hash, CacheEntryArgs *args);
```

Comment:
You're right. I hadn't submitted a new change until I was sure about the comment above. Sorry about that.